### PR TITLE
Own query for hebrew and improve default.sparql

### PR DESCRIPTION
**Pull request by dpriskorn**

### CODE REVIEW

**Comment by dpriskorn** [comment]
@Nudin does this look good? The default query returns 19361 results in 36365 ms :) Way below the limit.
Hebrew has gone from 7000 matches to 1400 so I hope that is a big improvement.

File: queries/he.sparql [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  FILTER (!regex(?desc, '.*theory.*')) .
+  FILTER (!regex(?desc, '.*mathemat.*')) .
+  FILTER (!regex(?desc, '.*geolog.*')) .
+  FILTER (!regex(?desc, '.*anatomy.*')) .
+  FILTER (!regex(?desc, '.*military.*')) .
+  FILTER (!regex(?desc, '.*taxonomic.*')) .
+  FILTER (!regex(?desc, '.*C++.*')) .
+  FILTER (!regex(?desc, '.*Unix.*')) .
+  FILTER (!regex(?desc, '.* law .*')) .
+  FILTER (!regex(?desc, '.*heraldic.*')) .
```
Comment (Nudin) [inline queries/he.sparql]: These don't make sense since the description is Hebrew – right?

**Review by Nudin — COMMENTED** [review]
Otherwise, seems good.

File: queries/he.sparql (hunk 1 above)
Comment (dpriskorn) [reply queries/he.sparql]: good catch!

**Pull request merged**